packages/block-editor/src/components/block-patterns-list/index.js

```diff
@@ -77,7 +79,12 @@ function BlockPattern( {
 		>
 			{ ( { draggable, onDragStart, onDragEnd } ) => (
 				<div
-					className="block-editor-block-patterns-list__list-item"
+					className={ clsx(
+						'block-editor-block-patterns-list__list-item',
+						{ 'is-active': isActive } // Apply 'is-active' class if this pattern is active
```

Comment:
I think we should separate the concepts of "active" and "selected":

- active, in this context, refers to the active composite item, ie,. the composite item that has focus when the parent composite widget is focused;
- selected, which is, I believe, the new feature that this PR wants to add (ie. the pattern that is being used)

If we were to continue with the current approach, I'd rename the variable to `selected`

Reply:
@ciampo Thanks for the clarification.

I will rename the active to selected in the above mentioned code snippets.

Along with that, I will work on some updates which will handle the styling of active element that has focus when the parent composite widget is focused.

Reply:
- Renamed the active to selected ignorer to handle the selected pattern.
- Changes applied to focus the active element while navigating through keyboard.